openstack/sfs_turbo/v1/shares/requests.go

```diff
@@ -84,6 +88,52 @@ func List(c *golangsdk.ServiceClient) ([]Turbo, error) {
 	return ExtractTurbos(pages)
 }
 
+// ListOptsBuilder allows extensions to add additional parameters to the List
+// request.
+type ListOptsBuilder interface {
+	ToVolumeListQuery() (string, error)
+}
+
+// ListOpts holds options for listing Volumes. It is passed to the volumes.List
+// function.
+type ListOpts struct {
+	// Requests a page size of items.
+	Limit int `q:"limit"`
+	// Used in conjunction with limit to return a slice of items.
+	Offset int `q:"offset"`
+}
+
+// ToVolumeListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToVolumeListQuery() (string, error) {
```

Comment:
it's better to rename the function name

Reply:
Here refer to the naming habits of other resources. Maybe they should have a uniform style